server/src/main/java/org/spine3/server/event/enrich/EnrichmentFunction.java

```diff
@@ -79,12 +80,37 @@ void setContext(EventContext context) {
     protected abstract Function<S, T> getFunction();
 
     /**
-     * Validates the instance.
+     * Activates the function.
+     *
+     * <p>During the activation the internal state of the function may be adjusted.
```

Comment:
Can you expand on this, please? It's not clear why the state needs to be changed. Perhaps an example would help.